plugins/github-deployments/README.md

```diff
@@ -0,0 +1,67 @@
+# GitHub Deployments Plugin
+
+The GitHub Deployments Plugin displays recent deployments from GitHub.
+
+![github-deployments-card](./docs/github-deployments-card.png)
+
+## Getting Started
+
+1. Provide OAuth credentials:
+
+- Create an [OAuth App](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/) and set env variables for ID and Secret.
+
+```bash
+export AUTH_GITHUB_CLIENT_ID={{YOUR_CLIENT_ID}}
+export AUTH_CLIENT_SECRET={{YOUR_CLIENT_SECRET}}
+```
+
+2. Install the GitHub Deployments Plugin.
+
+```bash
+# packages/app
+
+yarn add @backstage/plugin-github-deployments
+```
+
+3. Add the plugin to the app
+
+```typescript
+// packages/app/src/plugins.ts
+
+export { plugin as GithubDeployments } from '@backstage/plugin-github-deployments';
```

Comment:
but again, these packages should already have the correct name exported in the first place, ideally